source/clear-linux/guides/maintenance/time.rst

```diff
@@ -5,57 +5,58 @@ Set the time
 
 |CLOSIA| uses the `systemd-timesyncd.service` daemon to synchronize time.
 
-This section provides instructions to reset the time in your |CL| system when
+This guide describes how to reset the time in your |CL| system when
 the default :abbr:`NTP (Network Time Protocol)` servers cannot be reached.
 
-#. Install the `sysadmin-basic` bundle:
+#. Install the `sysadmin-basic` bundle.
 
-   .. code-block:: console
+   .. code-block:: bash
 
       swupd bundle-add sysadmin-basic
```

Comment:
change to "$ sudo swupd bundle-add sysadmin-basic"

Reply:
Partially completed: changed to "sudo swupd bundle-add sysadmin-basic"
If I add $ prompt, then users can't do a simple copy/paste.